src/functions.js

```diff
@@ -583,6 +587,9 @@ defineFunction(["\\over", "\\choose"], {
         case "\\choose":
             replaceWith = "\\binom";
             break;
+        case "\\atop":
+            replaceWith = "\\\\atopfrac";
+            break;
```

Comment:
Historically we've asked that PRs for new commands come with screenshot tests.  I'm sure there's lots of functionality that we don't have screenshot tests for.  I feel like we should have more screenshot tests so that we can have more comprehensive coverage.  I'd love to hear your thoughts on testing.